### ISSUE TITLE
problem: event dates for upcoming events are hard to parse

### ISSUE THREAD

**Issue by gazhayes** [EmurgoHK/cardanoupdate#317]
problem: event dates for upcoming events are hard to parse

<img width="317" alt="screenshot 2018-10-30 at 2 18 09 pm" src="https://user-images.githubusercontent.com/8143945/47699814-16550d80-dc4f-11e8-8048-7b32d01ecce4.png">

solution: use momentjs to make the dates for *upcoming* events very easy to read, e.g. `starts in 3 hours`. I guess the cutoff point is less than 48 hours, if it's more than 48 hours it's better to use the actual date.

Whoever takes this, while you're in there, the display tiles on the events page and home route shouldn't have the end date.

**Comment by geekyd** [comment]
@emurgobot claim

**Comment by geekyd** [comment]
@MadanBhandari Sure :)

**Comment by geekyd** [comment]
@MadanBhandari I'm unable to reproduce the issue, do you know how to reproduce it ?

**Comment by MadanBhandari** [comment]
It gives me January 1, 1970 5:30 AM date for everything :( Just add new event and go back to `/events` and check newly created event card

**Comment by geekyd** [comment]
That's strange. It works for me. However the event edit has stopped working for me after adding the daterangepicker. Fixing the edit functionality
